macos/generate_wazuh_packages.sh

```diff
@@ -41,6 +43,11 @@ function clean_and_exit() {
     exit ${exit_code}
 }
 
+ctrl_c() {
+    clean_and_exit 0
```

Comment:
Killing the process with a SIGKILL must return anything distinct from 0.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/45c5eb5ce3b01efdc2ac7be544d243b58e476832.